reana/cli.py

```diff
@@ -261,7 +257,7 @@ def cli():  # noqa: D301
         $ # once it works, we can tag and push
         $ reana-dev docker-build -t 0.3.0.dev20180625
         $ reana-dev docker-push -t 0.3.0.dev20180625
-        $ # we should now make PR for ``reana-cluster.yaml`` to use given tag
+        $ # we should now update ``reana-stable/values.yaml`` with the new tag
```

Comment:
`helm/reana/values.yaml` but you can remove this bit, rather say: "we should now publish stable helm charts for tag via chartpress" # FIXME